src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -896,6 +896,16 @@ def validate_params(self):
                 manifest.is_folder_allowed()
             ), f"Invalid value for `folder` {manifest.folder} must be one of {manifest.get_allowed_folders()}"
 
+    def create_single_custom_manifest(self, filename: str = None, folder: str = None, base64_content: str = None):
```

Comment:
why not reusing ``create_custom_manifests`` for the single-manifest purpose?
I don't think we should have a more specific function if there's a more generalized one that could be used instead

Let's instead introduce a new ``validate=True`` that allows skipping the validation if it's not needed

Reply:
Hi Osher Thanks for the review ,

I though about the option too , however additionally in our tests we want to add a lot of single custom manifests (one by one) , do some validations  in between , in using the general one this would mean that we will have to keep modifying the _config object of the cluster , and rerunning the create_custom_manifest on all in _config contained manifests .

Reply:
done